spec/features/bucket_lists/user_creates_bucket_list_spec.rb

```diff
@@ -15,17 +15,17 @@
 #x[] I recieve a success message when I successfully create a list
 #[x] I must be signed in to create the list
 
-scenario "authenticated user creates a bucket list" do
-  visit new_user_session_path
-  sign_in
-  visit root_path
-  click_link 'New Bucket List!'
-  fill_in 'Title', with: 'Europe'
-  click_button 'Create Your Bucket List!'
-
-  expect(page).to have_content("Congrats! You started a new Bucket List!")
-  expect(page).to have_content('Europe')
-end
+  scenario "authenticated user successfully creates a bucket list" do
+    visit new_user_session_path
+    sign_in
+    visit root_path
+    click_link 'New Bucket List!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.